product_docs/docs/biganimal/release/overview/02_high_availability/distributed_highavailability.mdx

```diff
@@ -57,6 +57,12 @@ To guard against cloud service provider failures, you can designate a witness no
 
 Cross-cloud service provider witness nodes are available with AWS, Azure, and Google Cloud using your own cloud account and BigAnimal's cloud account. This option is enabled by default and applies to both multi-region configurations available with PGD. For witness nodes you pay only for the infrastructure used, which is reflected in the pricing estimate.
 
+## Read-only workloads
+
+When you enable read-only workloads, a read-only connection string is created. You can use this connection to route read-only requests through the shadow nodes (non-write leaders) to lighten the load on the write leaders. In a two-node cluster, the single shadow node serves read-only workloads. In a three-node cluster, the two shadow nodes serve read-only workloads, unless a witness node is enabled. In a three-node cluster with a witness, read-only workloads are served by the single shadow node only.
+
+Similarly, if you have three data groups, the witness group will not serve any data requests. You can choose whether to enable read-only connections on the other two data groups, or just on one.
```

Comment:
@neel5481  Do we want to specifically mention "multi-region" for the multiple data groups setup? I was not sure if "multi-region" necessarily meant "more than one data group", hence why I tried to formulate it in a generic way here. 

However, if more than one data group means multi-region, I would change it here to be more specific. :)

Reply:
We can simplify this section by calling out that in any environment with 2 or 3 data nodes, the shadow node will perform read-only operations. 

And if there are more than one data groups, the customer can choose whether to enable the read-only connection on all data groups or none. 

In any case, witness nodes are not eligible.

Reply:
Not quite.

The eligible nodes (ie not the write leader, not a witness...) form a pool of read-only nodes
and read-only requests are round robinned around the pool.

Also, this feature will change when subscriber only nodes get added to the read-only pool at some point in the future. Probably best to discuss this in generalities.

There is the write leader where read-write queries should go, and the cluster selects a pool of eligible read-only nodes which will get the read-only traffic in a round robin fashion. If there is only one data node in the pool (e.g. a two data node and one witness group), then the read-only traffic will go to that node.

Also, you can enable read-only traffic per group, it's not all or nothing.

Reply:
@gvasquezvargas below is right

> "multi-region" necessarily meant "more than one data group"

Reply:
Correct on BA where it's a group per region, not correct for PGD where groups can be anywhere you like. (Points under the desk to a 3-group cluster in the same Docker environment). As we expect readers to move between the BA and PGD docs, we need to keep it reasonably interchangable. 

"Currently on Cloud Service, data groups in DHA clusters are assigned to different regions."